test.js

```diff
@@ -11,7 +11,6 @@ test.beforeEach(() => {
 
 test('translate without any options', async t => {
     const res = await translate('vertaler');
-
```

Comment:
Could you also keep original newlines. This is easier for reading tests.

Reply:
This has already been done.